app/controllers/middleware_server_controller.rb

```diff
@@ -141,6 +141,16 @@ def add_jdbc_driver
     }
   end
 
+  def jdbc_drivers
+    mw_server = MiddlewareServer.find(params[:server_id])
```

Comment:
missing `from_cid` around `params[:server_id]` - moving from step 1 to step 2, I get a `Couldn&#39;t find MiddlewareServer with &#39;id&#39;=10r2 [middleware_server/jdbc_drivers]` (a 500 response to `/middleware_server/jdbc_drivers?server_id=10r2`)